src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -681,13 +700,29 @@ void ensureLocationIndex(const std::string& tenant)
     std::string index = ENT_LOCATION "." ENT_LOCATION_COORDS;
     std::string err;
 
-    collectionCreateIndex(getEntitiesCollectionName(tenant), BSON(index << "2dsphere"), &err);
+    collectionCreateIndex(getEntitiesCollectionName(tenant), BSON(index << "2dsphere"), false, &err);
     LM_T(LmtMongo, ("ensuring 2dsphere index on %s (tenant %s)", index.c_str(), tenant.c_str()));
   }
 }
 
 
 
+/* ****************************************************************************
+ *
+ * ensureDateExpirationIndex -
+ */
+void ensureDateExpirationIndex(const std::string& tenant)
+{
+  /* Ensure index for entity expiration, in the case of using 2.4 */
+  if (mongoExpirationCapable())
+  {
+    std::string index = ENT_EXPIRATION;
+    std::string err;
+
+    collectionCreateIndex(getEntitiesCollectionName(tenant), BSON(index << 1), true, &err);
+    LM_T(LmtMongo, ("ensuring 2dsphere index on %s (tenant %s)", index.c_str(), tenant.c_str()));
```

Comment:
Wrong debug message... 2dsphere -> TTL (or something like that)

Reply:
Fixed in 5a4703d